modules/acquia_lift_publisher/acquia_lift_publisher.module

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @file
+ * Drupal Module: Acquia Lift - Publisher.
+ *
+ * Acquia Content Hub Publisher exports content from your Drupal site to
+ * Content Hub services. This module alters the behavior of the default
+ * functionality.
+ */
+
+use Drupal\acquia_lift_publisher\Form\AcquiaLiftPublisherSettingsForm;
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Entity\EntityInterface;
+
+/**
+ * Implements hook_entity_insert().
+ */
+function acquia_lift_publisher_entity_insert(EntityInterface $entity) {
+  _acquia_lift_publisher_trigger_queue($entity);
+}
+
+/**
+ * Implements hook_entity_update().
+ */
+function acquia_lift_publisher_entity_update(EntityInterface $entity) {
+  _acquia_lift_publisher_trigger_queue($entity);
+}
```

Comment:
I see insert and update. What about delete?

Reply:
Does deletion involve the export queue? It is an automatic process if I'm not mistaken, so it would remove the content from the subscriber site as well, wouldn't it?

Reply:
I am not so sure. @EclipseGc to weigh in on this one? ^^

Reply:
I think this is accurate. A delete would automatically remove the entity from Plexus via the acquia_contenthub_publisher module and we are not concerned with the syndication side from a Lift perspective, but if the customer happens to be both a Personalization and Syndication customer, the delete webhook would also delete those entities at the subscriber sites.

Reply:
Right, 8.x-2.x delete is not queued, so we don't have to worry about it here. Resolving this thread.